tests/Keboola/DbExtractor/AbstractRedshiftTest.php

```diff
@@ -24,7 +25,7 @@ public function setUp()
         if (!defined('APP_NAME')) {
             define('APP_NAME', 'ex-db-redshift');
```

Comment:
Would also remove this `APP_NAME` constant. Perhaps define it in some class as public static const?

Reply:
How are we dealing with it in other db extractors? :)